src/Meilisearch/Index.Tasks.cs

```diff
@@ -22,9 +22,9 @@ public async Task<Result<IEnumerable<TaskInfo>>> GetTasksAsync(CancellationToken
         /// <param name="taskUid">Uid of the task.</param>
         /// <param name="cancellationToken">The cancellation token for this call.</param>
         /// <returns>Returns the tasks.</returns>
-        public async Task<TaskInfo> GetTaskAsync(int taskUid, CancellationToken cancellationToken = default)
+        public async Task<TaskRessource> GetTaskAsync(int taskUid, CancellationToken cancellationToken = default)
```

Comment:
It should be `TaskResource` :)
<img width="258" alt="image" src="https://user-images.githubusercontent.com/4116980/178594882-d0374397-1731-4c61-b387-38b5699fdfe6.png">

Reply:
Actually it should be `TaskResults`